src/CreateMiddleware/CreateMiddleware.php

```diff
@@ -55,9 +56,13 @@ public function process($class, $projectRoot = null)
         $content = str_replace(
             ['%namespace%', '%class%'],
             [$namespace, $class],
-            self::CLASS_SKELETON
+            $classSkeleton ?: self::CLASS_SKELETON
```

Comment:
We can remove it at all !
We can set default value for the param to `self::CLASS_SKELETON`:

```php
public function process($class, $projectRoot = null, $classSkeleton = self::CLASS_SKELETON)
```